### ISSUE TITLE
Implement auth process for non-SSO users

### ISSUE THREAD

**Issue by YujinGaya** [sparcs-kaist/new-ara-web#21]
- [ ] 비밀번호 찾기: rest-auth/password/reset/
- [ ] 비밀번호 찾기 확인: rest-auth/password/reset/confirm/
- [ ] 로그인: rest-auth/login/
- [ ] 로그아웃: rest-auth/logout/
- [ ] 비밀번호 변경: rest-auth/password/change/

**Comment by talkwithraon** [comment]
위의 것을 모두 구현해야 하는 것은 아닙니다. 우선순위가 높은 것부터 시간이 허락하는 한도 내에서 진행해보면 좋을 것 같아요.

**Comment by apple12347678** [comment]
마이그레이션이 사실상 포기 수순이고 rest-auth/ api는 노후화 등 이유로 그냥 제거했습니다. 일단 non SSO 사용자는 고려하지 않고 작업하겠습니다.

**Comment by apple12347678** [comment]
@jwoo0122 non-SSO 사용자를 받을 시나리오를 생각해보면, 현재로써는 admin(new ara sysop)이 직접 집어 넣는 경우밖에 없음. 따로 프론트엔드나 api 엔드포인트를 만들 필요가 없다고 생각됨. 추후 관리 주체가 달라지면 이들을 위한 프론트/api가 생길 수는 있겠지만... v1.2쯤 가서 생각합시다.

**Comment by jwoo0122** [comment]
사실 nonSSO 를 꼭 고려 해야 되나 싶기도 함. SPARCS SSO 는 KAIST 계정 꼭 없어도 가입 가능하지 않던가?

**Comment by apple12347678** [comment]
그 SSO 계정이 접근할 수 있는 서비스가 뉴아라 외에도 많으니 문제
